specification/resources/resource-manager/Microsoft.Resources/stable/2024-11-01/resources.json

```diff
@@ -6447,6 +6447,18 @@
             "type": "string",
             "description": "Resource ID"
           },
+          "deploymentId": {
+            "type": "string",
+            "description": "Deployment ID"
```

Comment:
Could you expand on these descriptions a bit? They will be used to generate comments & documentation, so they're used by customers to discover what these properties mean.

For example: "The resource id of the Deployment responsible for this resource change."